packages/ifpack2/src/Ifpack2_ContainerFactory_decl.hpp

```diff
@@ -73,7 +73,7 @@ struct ContainerFactoryEntryBase
 {
   virtual Teuchos::RCP<Ifpack2::Container<MatrixType>> build(
       const Teuchos::RCP<const MatrixType>& A,
-      const Teuchos::Array<Teuchos::Array<typename MatrixType::local_ordinal_type>>& localRows,
+      const Teuchos::Array<Teuchos::Array<typename MatrixType::local_ordinal_type>>& partitions,
```

Comment:
You probably want this to be `ArrayView<const ArrayView<const local_ordinal_type>>`.